dracut/30ignition/selinux-policy-persist

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Allows for auto-updating the SELinux base policy
+# while keeping the policy directory that's running
+# on the node read/write. e.g. allowing installation
+# of non-reboot-persistent custom SELinux modules.
+
+set -euxo pipefail
+
+# Remove pre-existing policies.
+rm /etc/selinux/mcs
+rm /var/lib/selinux
+
+# install policies from image. Making them read/write.
+cp -a /usr/lib/selinux/mcs /etc/selinux
```

Comment:
I think this will break on update, or? The policy and the OS need to be updated together.

Reply:
Understood now, this should run on every boot